### DISCUSSION TITLE
Demo code MIME type issue

### DISCUSSION THREAD

**Discussion by wrabit** [idom-team/idom#748] [Problem]
From the [demo](https://idom-docs.herokuapp.com/docs/guides/escape-hatches/javascript-components.html#dynamically-loaded-components), seems that a MIME type is not set on the returned file? (Chrome, Mac M1)

```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

127.0.0.1/:1 Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: http://127.0.0.1:8000/_docs/_api/modules/__from_template__/@material-ui/core@%5E5.0.0
```

**Comment by rmorshea** [comment]
Thanks for the report. Unfortunately I won't be able to fix this myself [until next month](https://github.com/idom-team/idom/discussions/708). I'll be sure to address this once I'm back.

**Comment by Archmonger** [comment]
Issue created
- #749
